src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -632,6 +635,10 @@ def ignore_stack_frame(stack_frame, symbolized):
   # properly cross-platform.
   normalized_stack_frame = stack_frame.replace('\\', '/')
 
+  # Ignore frames that exactly match the app name.
```

Comment:
This seems a little bit risky but I think it makes sense in most cases. I'm happy to land this with just the tcmalloc fixes if you think that seems better.

Reply:
Discussed offline, keeping this.